src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/property-info": "~2.8|3.0",
         "symfony/security": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
-        "symfony/validator": "~2.8|~3.0",
+        "symfony/validator": "~3.1",
```

Comment:
So the solution would be to keep both classes (new and old) and use the old one in the core libraries, so we can still properly support and test 2.8 and 3.0?